apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/page/content/section/SectionCreator.java

```diff
@@ -37,6 +46,35 @@ public CreateSectionResponse createSection(long pageId, Long userId, CreateSecti
 
     }
 
+
+    public DeleteSectionResponse deleteSection(DeleteSectionRequest request) {
+        try {
+            log.info("Deleting section");
+            Integer orderNbr = waUiMetaDataRepository.getOrderNumber(request.sectionId());
+            waUiMetaDataRepository.deleteById(request.sectionId());
+            waUiMetaDataRepository.updateOrderNumberByDecreasing(orderNbr, request.sectionId());
+            return new DeleteSectionResponse(request.sectionId(), "Section Deleted Successfully");
+        } catch(Exception exception) {
+            throw new DeleteSectionException(exception.toString());
```

Comment:
I don't believe you can delete a section if it has any questions in it. We would need to verify that the next orderNumber after the section is either: 
1. another section
2. a tab
3. null 

if one of those 3 is true, then we can do the delete, else throw a `DeleteSectionException`